arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/ImcFragment.java

```diff
@@ -136,6 +140,30 @@ public Trace trace(Trace trace) {
       subFragment.trace(trace, hub.state.stamps());
     }
 
+    if (childFrame != null) {
+      trace.pMiscCcrsStamp(childFrame.revertStamp()).pMiscCcsrFlag(childFrame.selfReverts());
+    }
+
     return trace;
   }
+
+  // TODO: The most natural thing would be to implement resolveAtContextEntry instead.
+  @Override
+  public void resolveAtContextReEntry(Hub hub, CallFrame frame) {
+    childFrame = hub.callStack().getById(frame.childFramesId().getLast());
```

Comment:
We log the child call frame at context re-entry. This could alternatively have been done at `immediateContextEntry`. The only reason for doing it at re-entry is that we are fairly certain that at that point the child context will have been properly defined.